### PR TITLE
T2-CEPH-11314-Backup and restore data using existing NFS backup tools

### PR DESCRIPTION
**Pull request by hkadam134**
Signed-off-by: Hemanth <hyelloji@redhat.com>

Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-2RN022

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/cephfs_utilsV1.py [hunk 1]
```diff
@@ -1662,7 +1621,47 @@ def wait_for_cmd_to_succeed(self, client, cmd, timeout=180, interval=5):
...
+    #             sudo=True,
+    #             cmd=f"dd if=/dev/zero of={mounting_dir}{client.node.hostname}_dd bs=100M "
+    #             f"count=5",
+    #             long_running=True,
+    #         )
+    #
+    #     io_tools = [dd, smallfile]
+    #     f = random.choice(io_tools)
+    #     f()
+
```
Comment (AmarnatReddy) [inline tests/cephfs/cephfs_utilsV1.py:1663]: Can we remove commented lines

File: tests/cephfs/cephfs_utilsV1.py [hunk 2]
```diff
@@ -1718,79 +1723,64 @@ def cephfs_nfs_mount(self, client, nfs_server, nfs_export, nfs_mount_dir):
         return rc
 
     @staticmethod
-    def nfs_ganesha_conf(node, nfs_client_name):
-        out, rc = node.exec_command(
```
Comment (AmarnatReddy) [inline tests/cephfs/cephfs_utilsV1.py:1722]: Why this method is removed?
I think this method is used in upgrade test file

File: tests/cephfs/cephfs_utilsV1.py (hunk 1 above)
Comment (hkadam134) [reply tests/cephfs/cephfs_utilsV1.py]: DOne

File: tests/cephfs/cephfs_utilsV1.py (hunk 2 above)
Comment (hkadam134) [reply tests/cephfs/cephfs_utilsV1.py]: I replaced my cephfs_utils file so it would have gone due to that.
It's fixed now.

**Review by AmarnatReddy — APPROVED** [review]
LGTM

**Pull request merged**